### DISCUSSION TITLE
Docker gotcha, might be good to document?

### DISCUSSION THREAD

**Discussion by telemix** [anvilistas/amoni#15] [Ideas]
This is not an Amoni issue per se, but I came across it when installing Amoni. Others might, too.

Make sure the user you run `amoni start` with is in the docker group, else it will complain about permissions communicating with the docker socket : 
```
sudo usermod -aG docker $USER
newgrp docker
```
This is on Ubuntu 22.04, so you may need to adjust for other OSs.

I'd be happy to add it to the docs if someone can show me how? Something to do with pulling a git, I believe ...

**Comment by meatballs** [comment]
There are several decent guides online which explain the process of making a contribution to a repo on github. Here are a couple to start with:

https://reflectoring.io/github-fork-and-pull/
https://towardsdev.com/pull-push-and-fork-a-beginners-guide-to-git-and-github-3d31ee5b32

For the change you're describing, you probably want to add a line or two to:

https://github.com/anvilistas/amoni/blob/main/docs/installation.rst
